bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToAATo4B/SUSY_TTH_TTToAll_HToAATo4B_M-MASS/SUSY_TTH_TTToAll_HToAATo4B_M-MASS_proc_card.dat

```diff
@@ -0,0 +1,12 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set gauge unitary
+set complex_mass_scheme False
+
+import model NMSSM --modelname
```

Comment:
Is the model 4FS or 5FS per default?

Reply:
Hi @agrohsje , sorry to take so long in following up.  I tried following up with some other H --> aa analyzers about this question, but got no response.  I'm not sure how to check whether the model is 4FS or 5FS by default.  Do you have any suggestions about how I can find out from the model files themselves?
http://feynrules.irmp.ucl.ac.be/attachment/wiki/NMSSM/p1_ufo.tgz

Reply:
I checked myself. It is 4FS.